### PR TITLE
Add more test cases about toString and equals

### PR DESCRIPTION
**Pull request by Ma-Xizi**

### CODE REVIEW

**Comment by Ma-Xizi** [comment]
I am adding more tests about toString and equals method to increase the test coverage :)

File: src/test/java/tutorease/address/logic/commands/DeleteContactCommandTest.java [hunk 1]
```diff
@@ -32,6 +34,41 @@ public void constructor_nullIndex_throwsNullPointerException() {
...
+
+        Index index1 = Index.fromZeroBased(1);
+        DeleteContactCommand deleteCommand1 = new DeleteContactCommand(index1);
+        DeleteContactCommand deleteCommand2 = new DeleteContactCommand(index1);
+
+        // Test different objects but same Index
+        assertEquals(deleteCommand1, deleteCommand2);
+
+        Index index2 = Index.fromZeroBased(2);
+        DeleteContactCommand deleteCommand11 = new DeleteContactCommand(index1);
```
Comment (ethantyh692) [inline src/test/java/tutorease/address/logic/commands/DeleteContactCommandTest.java:62]: Could perhaps consider changing the naming of the variables here to something more intuitive and readable. For eg. deleteCommandWithIndex1 and deleteCommandWithIndex2

**Review by ethantyh692 — COMMENTED** [review]
LGTM, few nits to fix

File: src/test/java/tutorease/address/logic/commands/DeleteContactCommandTest.java (hunk 1 above)
Comment (NorbertLoh) [reply src/test/java/tutorease/address/logic/commands/DeleteContactCommandTest.java:62]: Agreed. Other than that LGTM!

Comment (Chronoxy) [reply src/test/java/tutorease/address/logic/commands/DeleteContactCommandTest.java:62]: Good read on this. But otherwise LGTM 👍

**Review by Chronoxy — CHANGES_REQUESTED** [review]

File: src/test/java/tutorease/address/logic/commands/DeleteContactCommandTest.java (hunk 1 above)
Comment (Ma-Xizi) [reply src/test/java/tutorease/address/logic/commands/DeleteContactCommandTest.java:62]: Fixed:)

File: src/test/java/tutorease/address/logic/commands/DeleteContactCommandTest.java [hunk 2]
```diff
@@ -32,6 +34,41 @@ public void constructor_nullIndex_throwsNullPointerException() {
...
+
+        Index index1 = Index.fromZeroBased(1);
+        DeleteContactCommand deleteCommand1 = new DeleteContactCommand(index1);
+        DeleteContactCommand deleteCommand2 = new DeleteContactCommand(index1);
+
+        // Test different objects but same Index
+        assertEquals(deleteCommand1, deleteCommand2);
+
+        Index index2 = Index.fromZeroBased(2);
+        DeleteContactCommand deleteCommandWithIndex1 = new DeleteContactCommand(index1);
```
Comment (ethantyh692) [inline src/test/java/tutorease/address/logic/commands/DeleteContactCommandTest.java:62]: Much better

**Review by ethantyh692 — APPROVED** [review]
LGTM

**Pull request merged**